source/reference.txt

```diff
@@ -19,10 +19,13 @@ Reference
    Inclusion and exclusion filters to specify which databases and
    collections to include in sync. 
 
+:ref:`c2c-authentication`
+   Use Workload Identity Federation to authenticate with clusters.
+
 :ref:`c2c-oplog-sizing`
    Increase the size of the oplog on the source cluster for large data
    sets. 
-   
+
```

Comment:
Nit: cruft in the PR

Reply:
No, that isn't cruft. That's removal of blank space.